scriptmodules/ports/openbor.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+# 
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="openbor"
+rp_module_desc="OpenBOR - Beat 'em Up Game Engine"
+rp_module_menus="4+"
+rp_module_flags="!x11 !mali"
+
+function depends_openbor() {
+    getDepends libsdl1.2-dev libsdl-gfx1.2-dev libogg-dev libvorbisidec-dev libvorbis-dev libpng-dev libz-dev
+}
+
+function sources_openbor() {
+    gitPullOrClone "$md_build" https://github.com/rofl0r/openbor.git
+}
+
+function build_openbor() {
+    make clean
+    NO_GL=1 make
+    cd "$md_build/tools/borpak/"
+    ./build-linux.sh
+    md_ret_require="$md_build/OpenBOR"
+}
+
+function install_openbor() {
+    md_ret_files=(
+       'OpenBOR'
+       'tools/borpak/borpak'
+       'tools/unpack.sh'
+    )
+
+}
+
+function configure_openbor() {
+    mkRomDir "ports"
+    mkRomDir "ports/$md_id"
+    mkUserDir "$configdir/$md_id"
+
+    cat >"$md_inst/extract.sh" <<_EOF_
+#!/bin/bash
+
+mkdir /home/pi/RetroPie/roms/ports/openbor/original/
+mkdir /home/pi/RetroPie/roms/ports/openbor/original/borpak/
+mv /home/pi/RetroPie/roms/ports/openbor/*.pak /home/pi/RetroPie/roms/ports/openbor/original/
+cp /opt/retropie/ports/openbor/unpack.sh /home/pi/RetroPie/roms/ports/openbor/original/
+cp /opt/retropie/ports/openbor/borpak /home/pi/RetroPie/roms/ports/openbor/original/borpak/
+cd /home/pi/RetroPie/roms/ports/openbor/original/
```

Comment:
hardcoded paths need adjusting - retropie can be installed for other users etc - you can use $home / $rootdir  / $mdinst etc - also rather than hardcoding them in the script they could be passed as a script parameter